examples/hello_world_par/Cargo.toml

```diff
@@ -15,9 +15,16 @@ workspace = true
 eframe = { workspace = true, default-features = false, features = [
     # accesskit struggles with threading
     "default_fonts",
+    "wayland",
+    "x11",
     "wgpu",
 ] }
 env_logger = { version = "0.10", default-features = false, features = [
     "auto-color",
     "humantime",
 ] }
+# This is normally enabled by eframe/default, which is not being used here
+# because of accesskit, as mentioned above
+winit = { workspace = true, features = [
+    "default"
```

Comment:
The example works without this, but the window frame looks different, so I added it to make it match the other examples.

#### With winit/default

![Screenshot from 2024-06-20 20-12-44](https://github.com/emilk/egui/assets/43298013/f823cf0c-9344-4c67-bea2-02b5c85ec291)

#### Without winit/default

![Screenshot from 2024-06-20 20-13-41](https://github.com/emilk/egui/assets/43298013/4c2d82e0-0561-4e40-b026-dd6aea99674d)